GTG/gtk/browser/browser.py

```diff
@@ -117,7 +132,6 @@ def __init__(self, requester, vmanager):
         self.activetree.register_cllbck('node-deleted-inview',
                                         self._update_window_title)
         self._update_window_title()
-        vmanager.timer.connect('refresh', self.refresh_workview)
```

Comment:
Search for refresh_all_views functions, I'm calling it up here to refresh all panes. could that hold as a solution?